microsoft-service-bus/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -245,6 +245,29 @@ payload]]></ee:set-payload>
 </mule>
 ----
 
+== SSL Issues
+
+You must configure the truststore in insecure mode if you get one of the following errors when performing a connectivity test or starting a Mule app:
+
+* `SSL certificate error`
+* `General SSLEngine problem`
+
+To configure the truststore in insecure mode:
+
+. Open the global element configuration.
+. Navigate to the *Security* tab.
+. In the *Security* field, select `Edit inline`.
+. In the *Trust Store Configuration* section, select the *Insecure* checkbox.
+. Click *OK*.
+
+.The graphic shows how to configure the truststore in insecure mode.
+image::ms-service-bus-truststore-insecure.png[Insecure truststore configuration]
+
+The first item shows the where to select `Edit inline` and the second item shows where to select the *Insecure* checkbox.
```

Comment:
```suggestion
The first item shows where to select **Edit inline**, and the second item shows where to select the **Insecure** checkbox.
```

Reply:
Why two starts instead of one on each sied?